src/peer/datastore.js

```diff
@@ -37,7 +38,12 @@ class Datastore {
     }
 
     levelOpts.valueEncoding = valueCodec
-    this.db = Levelup(levelOpts)
+    const location = (options.location == null || options.location === '')
+      ? '/aleph/data-' + uuid.v4()
+      : options.location
```

Comment:
`const location = options.location || '/aleph/data-' + uuid.v4()`

Unless you want `number` 0, `NaN` and `false` to be valid locations?

(there's a moment of wondering if js messes up the precedence in this but it's actually OK)